src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.ts

```diff
@@ -80,4 +66,9 @@ export class ModelingExamSubmissionComponent extends ExamSubmissionComponent imp
     get isActive(): boolean {
         return this.exercise && (!this.exercise.dueDate || moment(this.exercise.dueDate).isSameOrAfter(moment()));
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    modelChanged(model: UMLModel) {
```

Comment:
Maybe simply name the unused variable `_`. Its a common convention :)

Reply:
because the output event gives the changed model, otherwise we have a compiler error in the html template